tests/unit/mesh/test_mesh.cpp

```diff
@@ -180,3 +180,71 @@ TEST_CASE("MakeSimplicial", "[Mesh]")
    // on the original mesh, but it doesn't happen for these test cases.
    REQUIRE(simplex_mesh.GetNE() == orig_mesh.GetNE()*factor);
 }
+
+TEST_CASE("MakeNurbs", "[Mesh]")
+{
+   Array<double> intervals;
+   intervals.Append(1);
+   intervals.Append(1);
+   intervals.Append(1);
+   Array<int> continuity;
+   continuity.Append(-1);
+   continuity.Append(1);
+   continuity.Append(1);
+   continuity.Append(-1);
+   {
+      const KnotVector kv(2, intervals, continuity);
+      REQUIRE(kv.GetNE() == 3);
+      REQUIRE(kv.GetNCP() == 5);
+      REQUIRE(kv.GetOrder() == 2);
+      REQUIRE(kv.Size() == 8);
+   }
+   {
+      const KnotVector kv(3, intervals, continuity);
+      REQUIRE(kv.GetNE() == 3);
+      REQUIRE(kv.GetNCP() == 8);
+      REQUIRE(kv.GetOrder() == 3);
+      REQUIRE(kv.Size() == 12);
+   }
+   const KnotVector kv(2, intervals, continuity);
+   Array<NURBSPatch *> patches;
+   Array<double> grev_pts;
+   grev_pts.Append(0);
```

Comment:
Since the function from the linked PR still isn't available in `master`, I just kept the hard-coded Greville points, but switched to the initializer-list `Array` constructor you pointed to below.